modules/integration_azure-functions/conf/01-function-errors.yaml

```diff
@@ -0,0 +1,15 @@
+
+module: Azure Functions
+name: "wrapper errors"
+id: errors
+signals:
+  signal:
+    metric: azure.function.errors
+    extrapolation: 0
+rules:
+  critical:
+    threshold: 0
```

Comment:
Instead having a static threshold, we should compare the numbers of errors with the number of invocations.

ie having at least 10% of errors for 30miutes (to discuss)

Reply:
it depend the use case. For example a function that need to send data periodically, in case of function errors you will want to know immediately that your data were not sent to take corrective actions.

But if your function is a backend for a webapp, triggering an alert on each error would be to much.

Reply:
Good idea. 10% of errors for 30minutes as a default value seems good. This is overridable if needed.

Reply:
Done by using the same idea as on aws lambda detectors